### DISCUSSION TITLE
Importing one selected Skin from Stable to Lazer

### DISCUSSION THREAD

**Discussion by JofoxTheCat** [ppy/osu#21021] [Ideas] [answered]
My suggestion is (just as the title says) to select a/the Skin you want to import into osu!Lazer. That is really helpful if you have many skins and you don't want to wait like 5 mins everytime.... Thats basicly it. Thank you for reading :)

**Comment by WindowsMeosu** [comment, marked as answer]
So instead of importing all skins all once... you can import the skin manually from stable?

**Reply by peppy** [reply]
Yep, just export to `.osz` then drag into the lazer window.
